src/openforms/submissions/tasks/co_sign.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from django.conf import settings
+from django.utils.translation import ugettext
+
+from openforms.celery import app
+from openforms.emails.utils import render_email_template, send_mail_html
+from openforms.logging import logevent
+from openforms.submissions.models import Submission
+
+logger = logging.getLogger(__name__)
+
+__all__ = ["send_email_cosigner"]
+
+
+@app.task()
+def send_email_cosigner(submission_id: int) -> None:
+    submission = Submission.objects.get(id=submission_id)
+
+    recipient = submission.get_cosigner_email()
+    if not recipient:
+        logger.warning(
+            "No co-signer email found in the form. Skipping co-sign email for submission %d.",
+            submission.id,
+        )
+        return
+
+    # Send Co-sign email # TODO add configurable templates
+    content = render_email_template(
+        ugettext(
+            'This is a request to co-sign form "{{ form_name }}". Please go to the webpage of this form on our '
+            "website and click on the 'co-sign' button. You will then be redirected to authenticate yourself."
+            "After authentication, fill in the following code to retrieve the form submission:\n\n{{ code }}\n\n."
+        ),
+        context={
+            "code": submission.public_registration_reference,
+            "form_name": submission.form.name,
+        },
+    )
+
+    try:
+        send_mail_html(
+            subject=f"Co-sign request for {submission.form.name}",
+            html_body=content,
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            recipient_list=[recipient],
+            text_message=content,
+        )
+    except Exception:
+        logevent.cosigner_email_failure(submission)
+        raise
```

Comment:
this doesn't work how you expect it - because we use django-yubin, any `send_email*` calls actually just queue the e-mail which will be effectively sent via celery beat and another celery task.

So any SMTP borking will not show up here, and the logevent will never report failure even though it may fail.

I think logging SMTP failures is a whole different topic/task and should probably be removed from this PR.

Reply:
Do you mean that the all the log events here should be removed?
I had been following how the confirmation email was doing it, should we remove it from there too?
```
    try:
        send_mail_html(
            subject,
            html_content,
            settings.DEFAULT_FROM_EMAIL,  # TODO: add config option to specify sender e-mail
            to_emails,
            text_message=text_content,
        )
    except Exception as e:
        logevent.confirmation_email_failure(submission, e)
        raise

    submission.confirmation_email_sent = True
    ```

Reply:
Hrm, in this case it's probably just logging that the confirmation e-mail was scheduled correctly, so if we do the same for the cosign email, that's at least consistent. However, the wording of those log events is not correct as it doesn't prove that the e-mail was effectively sent successfully to an SMTP server :thinking: 

maybe `sent` should be reworded in the `scheduled` in the actual log messages? But ideally that should then be a separate issue + PR to keep the scope narrow here.